lapce-ui/src/terminal.rs

```diff
@@ -609,6 +609,7 @@ impl Widget<LapceTabData> for LapceTerminal {
                         term_data.receive_char(ctx, &s);
                     }
                 }
+                ctx.set_handled();
```

Comment:
This sits at the end of the "KeyPress" matcher and should be removed once the Terminal has a concept of "Input" and "Normal". For now, the terminal should just eat all the inputs. (Maybe? :D What do you want to do? This seemed like a good solution for the moment)

Reply:
The "Terminal" mode feels like a bit of a hack to me, but I'm not familiar with the underlying design decisions.